zha/zigbee/cluster_handlers/manufacturerspecific.py

```diff
@@ -475,8 +475,8 @@ async def async_update(self) -> None:
 
 
 @registries.CLUSTER_HANDLER_ONLY_CLUSTERS.register(IKEA_REMOTE_CLUSTER)
-@registries.CLUSTER_HANDLER_REGISTRY.register(IKEA_REMOTE_CLUSTER)
-class IkeaRemoteClusterHandler(ClusterHandler):
+@registries.CLIENT_CLUSTER_HANDLER_REGISTRY.register(IKEA_REMOTE_CLUSTER)
+class IkeaRemoteClientClusterHandler(ClusterHandler):
```

Comment:
Should this be a `ClientClusterHandler`? I guess it doesn't really need to be, as that only emits events IIRC